### DISCUSSION TITLE
How to rename GOOGLE SIGN IN URL this ?

### DISCUSSION THREAD

**Discussion by helloindra** [supabase/supabase#5055] [Questions] [answered]
Hello, how can i rename the Google Sign In supabase URL ? 

<img width="418" alt="image" src="https://user-images.githubusercontent.com/75231253/150123286-ab09ec90-f817-419b-8e58-4f8082c10934.png">

**Comment by silentworks** [comment, marked as answer]
Please try and use the search feature on here to find out if something like this has been asked before. You can find the answer in this thread https://github.com/supabase/supabase/discussions/2925#discussioncomment-1716295
